tools/test_configs/__init__.py

```diff
@@ -31,5 +35,11 @@ def get_default_config(target_name):
         if config_name == "NONE":
             return None
         return join(CONFIG_DIR, CONFIG_MAP[config_name])
+    elif target_name in TARGET_MAP:
+        if 'LWIP' in TARGET_MAP[target_name].features:
```

Comment:
Since the `else` following both the `elif` above and the `if` here have the same behavior, could this `if` be folded into the `elif`? for example:
```python
elif (target_name in TARGET_MAP and 
      'LWIP' in TARGET_MAP[target_name].features):
    return join(CONFIG_DIR, CONFIG_MAP["ETHERNET"])
```